storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -935,4 +936,21 @@ public static void setSubscriptionsFilter(SubscriptionsFilter value) {
     public static boolean shouldShowSubscriptionTitle() {
         return prefs.getBoolean(PREF_SUBSCRIPTION_TITLE, false);
     }
+
+    /**
+     * Returns whether should not show the RemoveAllInbox confirmation dialog.
+     * @return if should not show the remove all from inbox prompt dialog.
+     */
+    public static boolean shouldNotPromptRemoveAllFromInboxDialog() {
```

Comment:
I would actually move storage to the InboxFragment directly. UserPreferences is already quite huge and the setting only affects that one fragment. Similar to how it's done on the queue fragment